tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "compilerOptions": {
+    "outDir": "./es",
     "noLib": false,
-    "module": "commonjs",
-    "target": "es5",
+    "module": "es6",
```

Comment:
I believe that `ES2015` is the preferred terminology going forward. All newer targets for TypeScript are named in this manner.